Cargo.toml

```diff
@@ -19,20 +19,20 @@ version = "0.24.0"
 authors = ["The contributors to Trust-DNS"]
 edition = "2021"
 rust-version = "1.67.0"
-homepage = "https://trust-dns.org/"
-repository = "https://github.com/bluejekyll/trust-dns"
+homepage = "https://hickorydns.org/"
+repository = "https://github.com/hickorydns/hickory-dns"
```

Comment:
Is it feasible to get a `hickory-dns` domain instead of one without the dash? Same for the GitHub org... IMO the lack of consistency is unfortunate.

Reply:
I've already registered that. We can absolutely use the `-`, we can do that for Organization as well. I wasn't honestly sure which is better for folks to use. The other option is to remove the `-` from the project name to make it consistent?

Reply:
Personally I like having the `-`, visually separating words is pretty nice.

Reply:
ok, I updated.